tests/unit/app-template_test.cc

```diff
@@ -53,7 +52,11 @@ BOOST_AUTO_TEST_CASE(app_standard_memory_allocator,
     int actual_status = app.run(
         std::size(args), std::data(args),
         [expected_status] {
-            return make_ready_future<int>(expected_status);
+            // #2148 - add a small sleep to ensure the reactor does
+            // some of its background stuff, pollers etc for example.
+            return seastar::sleep(2s).then([expected_status] {
+                return make_ready_future<int>(expected_status);
```

Comment:
New flaky test!

Reply:
Should not be, as we will crash as soon as pollers are invoked. Considered waiting for something, but that is just the same thing. Need to ensure we simply end up in task queue, and a reactor poll is done.

Reply:
i understand that pollers are also queued like other tasks. but why does it matter? what exactly are we waiting? what happens if we do not wait long enough? could you please elaborate it a little bit?

Reply:
Pollers are run when we have nothing to run/after a task iteration. Even 2s here is probably on the high side. Just as long as we're pushed into the task queue, we should provoke the issue.

Reply:
I've updated the comment to be more explicit about what we try to do.